### DISCUSSION TITLE
Airflow DAG changes not reflected in the Airflow web UI and logs are not updated

### DISCUSSION THREAD

**Discussion by roy-sub** [apache/airflow#34671] [Q&A]
**Context -**

I'm facing an issue with `Apache Airflow` where `changes` I make to my `DAG` are not being reflected in the Airflow web UI, and the `logs` for my DAG are not updating as expected. However `DAGs` are getting triggered using the `REST API`. I'm not sure what's causing this problem, and I could use some help troubleshooting it.

**Additional Information -**

I initially set up `airflow` in my `GCP VM` and it was working fine. However when I `pushed ( migrated or transferred )` the code inside another `folder named ( say ‘etl’ )` I am facing the above mentioned issue.

I tried `re-initiating` the `airflow database - airflow db init`. However the changes or the logs are still not updated.

**Comment by Taragolis** [comment]
Let's start with general questions.

1. Which Airflow version are you use?
2. Deployment method? In docker in venv or other
3. Executor Type?
4. DB Type?
5. Are you use dynamic DAGs or any DAG factory?
6. Could you provide logs from the scheduler (service not a DAG) and DAG Processor (by default it stored into the`{AIRFLOW_HOME}/logs/dag_processor_manager/dag_processor_manager.log`)?
